package.json

```diff
@@ -114,13 +114,18 @@
     "mocha": "^9.2.0",
     "node-fetch": "^2.6.1",
     "postcss": "^8.2.4",
+    "postcss-html": "^1.5.0",
     "postcss-loader": "^4.2.0",
     "postcss-preset-env": "^6.7.0",
+    "postcss-scss": "^4.0.6",
```

Comment:
Is that package required? I tried removing it, and stylelint didn't complain.

Reply:
yes! Looks like its a double failing that works though : `post-css-scss` seems necessary to my config (testing by removing  `post-css-scss` and `stylelint-config-recommended-scss`), but `stylelint-config-recommended-scss` is bundling `post-css-scss`. That's why it's still working if you remove this line. I would be removing this line and for keeping  `stylelint-config-recommended-scss` as configuration does look fit to the codebase and bundles `postcss-scss`.

Reply:
line removed in 3591f6ef1